src/scss/_mixin.scss

```
@@ -75,6 +75,25 @@
   cursor: default;
 }
 
+// スクロール用マスク
+// (↓スクロールさせたい要素の親要素の疑似after/before要素に設定する。)
```

```suggestion
// (↓スクロールさせたい要素をwrapする親要素に設定する。)
```
これわたしがやばい方法でmaskにチャレンジしてたときに変え忘れたコメントなので、"commit suggestion"だけ押してついでに取り込んでいただけないでしょうか…!

---

いやなんか自分でできたので大丈夫です(